tsconfig.json

```diff
@@ -29,7 +29,7 @@
     ".next/types/**/*.ts",
     "**/*.ts",
     "**/*.tsx"
-  ],
+, "pages/index.tsx"  ],
```

Comment:
why was this changed?

Reply:
That updated automatically when I renamed index.js to index.tsx

Reply:
remove it? `**/*.tsx` covers it anyways.

Reply:
Done